backend/btrixcloud/basecrawls.py

```diff
@@ -209,7 +209,9 @@ async def delete_crawls(
 
         res = await self.crawls.delete_many(query)
 
-        quota_reached = await self.orgs.inc_org_bytes_stored(org.id, -size)
+        if not type_:
```

Comment:
Hm, i think maybe we can't do a bulk delete without type, since it could be a mixture of items?
Probably will need to do three deletes, so we know which quota to decrease?

Reply:
I reworked `delete_crawls_all_types` to call the `delete_crawls` endpoint separately for crawls and uploads, and added in a missing routine to update the associated workflows and recompute the last crawl statistics. That routine also had a bug where it assumed only one crawl was being deleted per cid, so fixed that as well.